test/component_test.go

```
@@ -141,6 +141,68 @@ func (s *ComponentSuite) TestBasic() {
+	runnersGVR := schema.GroupVersionResource{
+		Group:    "actions.summerwind.dev",
+		Version:  "v1alpha1",
+		Resource: "runners",
+	}
+
+	factory := dynamicinformer.NewFilteredDynamicSharedInformerFactory(dynamicClient, time.Minute, corev1.NamespaceAll, nil)
+	informer := factory.ForResource(runnersGVR).Informer()
+
+	stopChannel := make(chan struct{})
+
+	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

golangci-lint (1.64.8)

Error return value of `informer.AddEventHandler` is not checked

(errcheck)